beetmoverscript/script.py

```diff
@@ -56,6 +57,12 @@
     # validate scopes to prevent beetmoving in the wrong place
     validate_task_scopes(context, mapping_manifest)
 
+    # some files to-be-determined via script configs need to have their
+    # contents pretty named, so doing it here before even beetmoving begins
+    blobs = context.config.get('blobs_needing_prettynaming_contents', [])
+    if blobs:
```

Comment:
This check actually isn't needed.  If `blobs == []`, then the `for` loop in `alter_unpretty_constants` will be noop.